apps/prairielearn/src/lib/manualGrading.ts

```diff
@@ -301,9 +301,13 @@ export async function updateAssessmentQuestionRubric(
           const updated =
             item.id == null
               ? null
-              : await sqldb.queryOptionalRow(sql.update_rubric_item, item, IdSchema);
+              : await sqldb.queryOptionalRow(
```

Comment:
I know this is not the point of this PR, but as long as we're changing this, could we instead structure this like:

```ts
if (item.id) {
  // do update
} else {
  // do insert
}
```

This wouldn't handle the case where someone provides an illegitimate `id` from the client, but I'm extremely comfortable just dropping data if that happens, or even erroring out completely. cc @jonatanschroeder

Reply:
We had a conversation thread around this on #pl-dev

Reply:
Yes, I'm ok with that. Maybe change to `queryRow` so that an error happens in this scenario?

Reply:
Looks like this has been resolved 👍